app/controllers/revoked_controller.rb

```diff
@@ -1,13 +1,14 @@
 class RevokedController < ApplicationController
   def revoked_organisation
-    organisation = Organisation.find_by_organisation_id(params[:name])
+    ssa = Ssa.find_by_ssa_id(params[:ssa_id])
 
-    render json: { "revoked": organisation.revoked }
+    render json: { "revoked": ssa.revoked }
   end
 
   def revoked_software
-    ssa = Ssa.find_by_ssa_id(params[:ssa_id])
+    organisation = Organisation.find_by_organisation_id(params[:name])
+    cert = organisation.certificates.find { |cert| cert.usage == 'signing' }
```

Comment:
In case I forget, we need to extend this in some way for the transport certificates, at least. But I recognise this isn't meant to be a finished implementation of this controller.

Reply:
Yeah this is true. I ignored it this time round as we wouldn't necessarily be interacting with revocation endpoints in the short term